### PR TITLE
Blogpost/roc2

### PR DESCRIPTION
**Pull request by alanjschoen**
Please review my latest blog post.  It's the second part about measuring accuracy.

### CODE REVIEW

File: _posts/2017-05-10-Measuring_Performance_part2.md [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+published: True
+---
+
+This post is the second and final installment in my series about measuring the accuracy of models.  In my last post, I talked about how to choose a method to score models.  Now it's time to use that method to score models against each other.  Let's start with a brief summary of the goal.
+
+We have a ground truth image, in which each airplane has been painstakingly marked by an intern in the DigitalGlobe offices:
+
+![Airplane Graveyard]({{ site.baseurl }}/assets/images/Measuring_Performance_2/graveyard_marked.png){: width="80%"}
+
+A model's goal is to find all of the airplanes in this image.  Which would look like this:
```
Comment (geovations) [inline _posts/2017-05-10-Measuring_Performance_part2.md]: you may want to change that period to a comma and concat that last clause

File: _posts/2017-05-10-Measuring_Performance_part2.md [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+
+### Narrowing it down more: threshold 0.593
+I'm going to show one more variation that shows what happens when the threshold is too high.
+
+![Airplane Graveyard]({{ site.baseurl }}/assets/images/Measuring_Performance_2/prediction3.png){: width="65%"}
+&nbsp;&nbsp;&nbsp;
+![Airplane Graveyard]({{ site.baseurl }}/assets/images/Measuring_Performance_2/hist3.png){: width="22%"}
+
+At this threshold leve, we don't have any false positives, but now we're missing half of the planes.  We have only 16 true positives, out of a maximum of 33.  As a result, the model has an `F1` score of 0.65, which is lower than the last threshold.
```
Comment (geovations) [inline _posts/2017-05-10-Measuring_Performance_part2.md]: I think you meant *level* there.

Comment (alanjschoen) [inline _posts/2017-05-10-Measuring_Performance_part2.md]: My new commit addresses these issues

Comment (alanjschoen) [inline _posts/2017-05-10-Measuring_Performance_part2.md]: Fixed in new commit

File: _posts/2017-05-10-Measuring_Performance_part2.md (hunk 1 above)
Comment (alanjschoen) [inline _posts/2017-05-10-Measuring_Performance_part2.md]: Rephrased in new commit

**Pull request merged**